.github/workflows/daily-runtime-validation.yml

```diff
@@ -44,6 +44,18 @@ jobs:
           make
           echo "$GITHUB_WORKSPACE/wasm-micro-runtime/product-mini/platforms/linux/build" >> $GITHUB_PATH
 
+      - name: Install WasmEdge
+        if: matrix.runtime == 'wasmedge'
+        run: |
+          sudo apt install -y software-properties-common cmake libboost-all-dev
+          sudo apt install -y clang
+          git clone https://github.com/WasmEdge/WasmEdge.git
+          cd WasmEdge
+          mkdir -p build && cd build
+          cmake -DWASMEDGE_BUILD_AOT_RUNTIME=OFF .. && make -j
+          echo "$GITHUB_WORKSPACE/WasmEdge/build/tools/wasmedge" >> $GITHUB_PATH
+          cp .github/scripts/wasi-testsuite-adapter.py ../adapters/wasmedge.py
```

Comment:
does this mean to test with the latest (unreleased) version of wasmedge?

Reply:
Yes